lib/matplotlib/widgets.py

```diff
@@ -702,14 +702,16 @@ def __init__(
             f'marker{k}': v for k, v in {**defaults, **handle_style}.items()
         }
 
+        verts = self._fill_verts(np.zeros([5, 2]), valinit)
+
```

Comment:
```suggestion
```
Need to move this after the Polygon creation to be able to use the high-level interface.